client/src/components/RegisterPage/RegisterForm.jsx

```diff
@@ -88,7 +93,11 @@ const RegisterForm = () => {
                     error={errors.password?.message}
                     {...register('password')}
                 />
-                <FormButton type="submit" disabled={Object.keys(errors) > 0} isLoading={isLoading}>
+                <FormButton
+                    type="submit"
+                    disabled={Object.keys(errors) > 0}
```

Comment:
is this meant to be `Object.keys(errors).length`?
```suggestion
                    disabled={Object.keys(errors).length > 0}
```